app/shared_config_manager/services.py

```diff
@@ -191,3 +197,148 @@ def _proc_iter(proc: subprocess.Popen[bytes]) -> Iterable[Union[bytes, Any]]:
         yield block
     if proc.wait() != 0:
         raise HTTPServerError("Error building the tarball")
+
+
+@_ui_index_service.get(renderer="./templates/index.html.mako")  # type: ignore
```

Comment:
Cornice is for REST API
And maybe move to views.py or views folder.
And add tests.